packages/flutter/lib/src/material/radio.dart

```diff
@@ -155,6 +157,69 @@ class Radio<T> extends StatefulWidget {
   /// ```
   final ValueChanged<T> onChanged;
 
+  /// Set to true if this radio button is allowed to be returned to an
+  /// indeterminate state by selecting it again when selected.
+  ///
+  /// To indicate returning to an indeterminate state, [onChanged] will be
+  /// called with null.
+  ///
+  /// If true, [onChanged] can be called with [value] when selected while
+  /// [groupValue] != [value], or with null when selected again while
+  /// [groupValue] == [value].
+  ///
+  /// If false, [onChanged] will be called with [value] when it is selected
+  /// while [groupValue] != [value], and only by selecting another radio button
+  /// in the group (i.e. changing the value of [groupValue]) can this radio
+  /// button be unselected.
+  ///
+  /// The default is false.
+  ///
+  /// {@tool dartpad --template=stateful_widget_scaffold}
+  /// This example shows how a radio button can be allowed to be toggled off by
```

Comment:
shows how to enable deselecting a radio button by setting the...

Reply:
Done.